proto/attestation/reference_value.proto

```diff
@@ -139,17 +157,15 @@ message KernelLayerReferenceValues {
   // Verifies the kernel based on endorsement.
   KernelBinaryReferenceValue kernel = 1;
 
-  // No longer used. Will be removed.
-  // TODO: b/325979696 - Validate the kernel command-line using a regex.
-  BinaryReferenceValue kernel_cmd_line = 2 [deprecated = true];
-
-  // Validates the kernel command-line using a regex.
-  RegexReferenceValue kernel_cmd_line_regex = 8;
+  // Validates the kernel command-line using either a string literal or a regex.
+  TextReferenceValue kernel_cmd_line_text = 9;
 
   // Fields are deprecated and kept only for backwards compatibility.
```

Comment:
Maybe we should clarify that these fields are not just kept for compatibility, but also being completely ignored now by the verifier. 

And maybe for good measure the verifier should actually fail if any of these values are still being specified. That would avoid all confusion about the meaning of these fields in cases where someone erroneously specifies both the new and old values.

Reply:
Failing verification sounds scary at this point but expanded the comment